Userland/Applications/Browser/WebDriverConnection.cpp

```diff
@@ -129,7 +129,7 @@ Messages::WebDriverSessionClient::GetDocumentElementResponse WebDriverConnection
         if (tab.on_get_document_element)
             return { tab.on_get_document_element() };
     }
-    return { {} };
+    return Optional<i32> {};
```

Comment:
Don't think we should do this, should never be reached in a proper set-up anyway